cmd/gaia/app/app.go

```diff
@@ -40,7 +40,8 @@ type GaiaApp struct {
 	*bam.BaseApp
 	cdc *codec.Codec
 
-	assertInvariantsBlockly bool
+	nextInvariantsCheck       uint
```

Comment:
What is `nextInvariantsCheck` for? We don't need it imho.

Reply:
agreed, I'm dropping it